app/Statistics/Repository/IndividualRepository.php

```diff
@@ -642,6 +643,7 @@ public function commonSurnamesListTotals(
     public function statsBirthQuery(bool $sex = false, int $year1 = -1, int $year2 = -1): array
     {
         $query = DB::table('dates')
+            ->select(['d_month', DB::raw('COUNT(*) AS total')])
```

Comment:
Do these changes fix a problem - or do they just remove the `else`?

Reply:
removes teh else